projects/jdwp/BUCK

```diff
@@ -0,0 +1,11 @@
+jdwp_binary(
```

Comment:
Buck does not know what `jdwp_binary` is, but you can tell it to build parts of python project with the following rules:
- [`python_library`](https://github.com/facebook/buck2/blob/main/examples/with_prelude/python/library/BUCK) 
- [`python_binary`](https://github.com/facebook/buck2/blob/main/examples/with_prelude/python/hello_world/BUCK#L3C1-L3C14)
- `python_test` (see "Examples" [here](https://buck.build/rule/python_test.html))

Reply:
I have made some changes, and the targets run without errors or any blocker